src/tests/topp/FeatureFinderMetabo_2_noEPD_output.featureXML

```diff
@@ -5,14 +5,14 @@
 		<processingAction name="Quantitation" />
 		<userParam type="string" name="parameter: mode" value="test_mode"/>
 	</dataProcessing>
-	<featureList count="16">
+	<featureList count="13">

```

Comment:
actually, only the RT filtering is disabled (instead of the old workaround for non-LC data). That\s why three more traces are combined.
Some parameters changes from before are irrelevant, the current values are the default.
Making an extra test does not make sense IMHO, since the current setting is the one that fits the data. Sorry if that led to confusion.